docs/en/ingest-management/elastic-agent/configuration/elastic-agent-monitoring.asciidoc

```diff
@@ -38,6 +38,12 @@ To enable monitoring, set `agent.monitoring.enabled` to `true`. Also set the
 collected. If neither setting is specified, monitoring is turned off. Set
 `use_output` to specify the output to which monitoring events are sent.
 
+Setting `agent.monitoring.enabled` to `true` also enables a `/liveness` endpoint to be 
+available where that server is configured to be exposed. By default, the endpoint 
+returns a `200` OK status as long as {agent}'s internal main loop is responsive and can 
+process configuration changes. It can be configured to also monitor the component states 
+to return an error if anything is degraded or failed.
```

Comment:
"where that server is configured to be exposed" is a little confusing to me - does that just mean at the URL for the server? Would that be different for `/debug/pprof/` and `/buffer`? if not, I'd probably skip the "where configured" bit

```suggestion
Setting `agent.monitoring.enabled` to `true` also exposes a `/liveness` endpoint. By default, the endpoint 
returns a `200` OK status as long as {agent}'s internal main loop is responsive and can 
process configuration changes. It can be configured to also monitor the component states 
and return an error if anything is degraded or has failed.
```